crates/biome_rule_options/src/no_restricted_imports.rs

```diff
@@ -77,9 +98,201 @@ pub struct CustomRestrictedImportOptions {
     pub allow_import_names: Box<[Box<str>]>,
 }
 
+impl PathOptions {
```

Comment:
I think we should avoid to put js-specific stuff in the biome_rule_options crate if possible, as it will be shared among languages.

Reply:
Plus the rule implementation for a language should be colocated.